### PR TITLE
Adds {orgrepo} to replace refs to github.com

### LINKED ISSUE #30: Update references and config statements in conf.py

### PR DESCRIPTION
**Pull request by pedroalvesbatista**
Signed-off-by: Pedro Alves Batista <pedro.pesquisapb@gmail.com>

## Description of the Pull Request (PR):

Adds new {orgrepo} to replace refs to github.com.

## This fixes or addresses the following GitHub issues:

- Fixes #30

### CODE REVIEW

**Comment by DrDaveD** [comment]
Actually, I'm sorry, I take it back: it's OK to use it in a github.com URL; there is only one place where it shouldn't be, and that's the first one listed in conf.py.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
Besides the conflict, I still see a few additional github.com references to singularityware/singularity and sylabs/singularity that should be updated to use `{orgrepo}`.  

You might as well also change hpcng/singularity in the epub_identifier variable in conf.py to apptainer/apptainer at this time (I'm not sure if that could use `{orgrepo}`, I'm guessing not).

**Comment by pedroalvesbatista** [comment]
There is also a example part in `gpu.rst` pointing to an example using Blender/OpenCL. Can we keep or look for some other way to reproduce this and put in out repo ?

**Comment by DrDaveD** [comment]
> There is also a example part in `gpu.rst` pointing to an example using Blender/OpenCL. Can we keep or look for some other way to reproduce this and put in out repo ?

I'm not sure, but in any case it's not part of this issue.  I suggest creating a separate issue and we'll consider it later.

File: appendix.rst [hunk 1]
```diff
@@ -794,9 +794,9 @@ Notes
 Arch Linux is, by design, a very stripped down, light-weight OS. You may
 need to perform a significant amount of configuration to get a usable
 OS. Please refer to this `README.md
-<https://github.com/singularityware/singularity/blob/master/examples/arch/README.md>`_
+<https://github.com/apptainer/apptainer/blob/master/examples/arch/README.md>`_
 and the `Arch Linux example
-<https://github.com/singularityware/singularity/blob/master/examples/arch/{Project}>`_
+<https://github.com/apptainer/apptainer/blob/master/examples/arch/>`_
```
Comment (DrDaveD) [inline appendix.rst:799]: These should be `{orgrepo}` instead of `apptainer/apptainer`.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: contributing.rst [hunk 2]
```diff
@@ -58,11 +58,11 @@ want to share the love so nobody feels left out!
...
 You can contribute to the documentation by `raising an issue to suggest
 an improvement
-<https://github.com/apptainer/apptainer-userdocs/issues/new>`_ or by
+<https://github.com/apptainer-userdocs/issues/new>`_ or by
 sending a `pull request
-<https://github.com/apptainer/apptainer-userdocs/compare>`_ on `our
+<https://github.com/apptainer-userdocs/compare>`_ on `our
 repository for documentation
-<https://github.com/apptainer/apptainer-userdocs>`_.
+<https://github.com/apptainer-userdocs>`_.
```
Comment (DrDaveD) [inline contributing.rst:65]: These should be `{orgrepo}` instead of `apptainer`.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

**Comment by DrDaveD** [comment]
I'll finish this one too so it's done

**Comment by DrDaveD** [comment]
Completed in #55

**Pull request closed without merging**